src/main/kotlin/me/partlysanestudios/partlysaneskies/features/discord/webhooks/Webhook.kt

```diff
@@ -0,0 +1,43 @@
+//
+// Written by Su386.
+// See LICENSE for copyright and license notices.
+//
+
+
+package me.partlysanestudios.partlysaneskies.features.discord.webhooks
+
+import gg.essential.elementa.UIComponent
+import me.partlysanestudios.partlysaneskies.config.psconfig.Config
+import me.partlysanestudios.partlysaneskies.config.psconfig.ConfigManager
+import me.partlysanestudios.partlysaneskies.config.psconfig.Toggle
+import me.partlysanestudios.partlysaneskies.config.psconfig.Toggle.Companion.asBoolean
+import me.partlysanestudios.partlysaneskies.config.psconfig.Toggle.Companion.asToggle
+
+abstract class Webhook(defaultEnabledState: Boolean = false) {
+    companion object {
+        private const val WEBHOOK_FOLDER_PATH = "webhooks/"
+    }
+    var enabled: Boolean get() {
```

Comment:
please have the get() = and set() {} in their own seperate line
use get() = instead of get() { return